Release Candidate version 2.6.3 (#81)
aburrell/aacgmv2: CONTRIBUTING.rst

       have trouble building all the testing environments, you can rely on

>        Travis and AppVeyor - they will run the tests for each change you add in

Author: ```suggestion
       GitHub Actions - it will run the tests for each change you add in
```
